src/js_types.hpp

```diff
@@ -126,6 +126,7 @@ struct Value {
     static bool is_boolean(ContextType, const ValueType &);
     static bool is_constructor(ContextType, const ValueType &);
     static bool is_date(ContextType, const ValueType &);
+    static bool is_error(ContextType, const ValueType &);
```

Comment:
whenever we change the abstraction layer we need to handle it both for Nodejs and RN